controllers/permits/applications/special-uses/postSchema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "region": {"default":0},
+  "forest": {"default":0},
+  "district": {"default":0},
+  "authorizingOfficerName": {"default":""},
+  "authorizingOfficerTitle": {"default":""},
+  "securityId": {"default":""},
+  "managingOrg": {"default":""},
+  "adminOrg": {"default":""},
+  "sudsId": {"default":""},
+  "effectiveDate": {"default":""},
+  "status": {"default":"Accepted"},
+  "applicantInfo": {
+    "firstName": {"default":""},
+    "lastName": {"default":""},
+    "dayPhone": {
+      "areaCode": {"default":0},
+      "number": {"default":0},
+      "extension": {"default":0},
+      "type": {"default":""}
+    },
+    "eveningPhone": {
+      "areaCode": {"default":0},
+      "number": {"default":0},
+      "extension": {"default":0},
+      "type": {"default":""}
+    },
+    "emailAddress": {"default":""},
+    "mailingAddress": {"default":""},
+    "mailingAddress2": {"default":""},
+    "mailingCity": {"default":""},
+    "mailingState": {"default":""},
+    "mailingZIP": {"default":0},
+    "organizationName": {"default":""},
+    "website": {"default":""},
+    "orgType": {"default":""},
+    "countryName": {"default":"United States of America"},
+    "contName": {"default":""},
+    "contactType": {"default":""},
+    "addressType": {"default":"Main"} 
+  },
+  "type": {"default":""},
+  "noncommercialFields": {
+    "activityDescription": {"default":""},
+    "locationDescription": {"default":""},
+    "startDateTime": {"default":""},
+    "endDateTime": {"default":""},
+    "numberParticipants": {"default":0},
+    "purpose": {"default":""},
+    "crExempt": {"default":"Y"},
+    "formName": {"default":"FS-2700-3b"}
+  },
+  "tempOutfitterFields": {
+    "activityDescription": {"default":""},
+    "locationDescription": {"default":""},
```

Comment:
Need to remove this, purpose is no longer required for tempOutfitters so no need for locationDescription, start/end DateTIme, or purpose.